postgres/problems/problem_04.md

```
@@ -1,20 +1,23 @@
-  * 集計値が同じ場合は `user_id` 昇順で並べて **tie‑break** してください
+* DISTINCT を使用すること
+* WHERE, ORDER BY, JOIN などの追加句は使用しないこと
 
 ## 想定出力例（先頭 5 行）
-
-| user_id | user_name | order_count |
-|---------|-----------|-------------|
-|     101 | Alice     |          37 |
-|     212 | Bob       |          35 |
-| …       | …         |         …  |
+ 
```

markdownlint-cli2 (0.17.2)

Trailing spaces
Expected: 0 or 2; Actual: 1

(MD009, no-trailing-spaces)